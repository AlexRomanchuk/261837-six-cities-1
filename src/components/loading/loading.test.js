import React from "react";
import renderer from "react-test-renderer";
import Loading from "../loading/loading.jsx";

it(`Loading page snapshot`, () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});
