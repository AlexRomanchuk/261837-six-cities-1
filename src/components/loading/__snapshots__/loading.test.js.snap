// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading page snapshot 1`] = `
<main
  className="page__main page__main--index"
>
  <div
    className="cities__places-wrapper"
  >
    <div
      className="cities__places-container container"
    >
      <section
        className="cities__places places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            Loading...
          </b>
          <p
            className="cities__status-description"
          >
            Data is loading from server.
          </p>
        </div>
      </section>
      <div
        className="cities__right-section"
      >
        <section
          className="cities__map map"
        />
      </div>
    </div>
  </div>
</main>
`;
